README.md

```diff
@@ -13,7 +13,11 @@
 
 This repository contains the source code for the official Vivaann Welfare Association website.
 
-> Designed & built by [Ajit Panigrahi](https://twitter.com/ajitzero). Hosted on [GitHub Pages](https://pages.github.com/).
+Version 1 Designed & built by [Ajit Panigrahi](https://twitter.com/ajitzero).
```

Comment:
Remove my name and add:

"Currently maintained and managed by [Abhiraj Singh Chauhan](https://www.linkedin.com/in/abhirajsinghchauhan/)"

Add your name in the release (I'll add that once this gets merged)